test/blackbox-tests/test-cases/dune-cache/trim.t

```diff
@@ -34,7 +34,7 @@ Function to reset build tree and cache.
 Check that trimming does not crash when the cache directory does not exist.
 
   $ dune cache trim --size 0B
-  Freed 0 bytes
+  Freed 0 B
```

Comment:
The space between 0 and B seems unnecessary.

Reply:
(We usually write just "5GB".)

Reply:
I've removed the space.